packages/zoe/src/zoeService/zoe.js

```diff
@@ -244,27 +244,23 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
           getIssuers: () => instanceRecord.terms.issuers,
           getBrands: () => instanceRecord.terms.brands,
           getInstance: () => instance,
-          hasShutdown: () => hasShutdown,
-          shutdown: () => {
-            hasShutdown = true;
+          acceptingOffers: () => acceptingOffers,
+          exitAllSeats: () => {
+            acceptingOffers = false;
             zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.exit());
           },
+          kickOutAllSeats: reason => {
+            acceptingOffers = false;
+            zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.kickOut(reason));
+          },
+          stopAcceptingOffers: () => (acceptingOffers = false),
         };
       };
 
       const instanceAdmin = makeInstanceAdmin();
-
       instanceToInstanceAdmin.init(instance, instanceAdmin);
 
-      E(adminNode)
-        .done()
-        .then(
-          () => instanceAdmin.shutdown(),
-          () => instanceAdmin.shutdown(),
```

Comment:
Did we lose the handling of unexpected vat shutdowns? Where is this pathway handled, and are the seats exited when this is the case? By this, I mean when `zcf.shutdown` and `zcf.shutdownWithError/Failure` are not called, but something wrong happens in the vat.

Reply:
Thank you. 

I had lost track of the fact that we had disabled all the metering tests and was counting on them to check this case.